msbuild/Xamarin.MacDev.Tasks.Core/Tasks/WriteAppManifestTaskBase.cs

```diff
@@ -0,0 +1,51 @@
+#nullable enable
+
+using System;
+using System.IO;
+using System.Linq;
+
+using Microsoft.Build.Framework;
+
+using Xamarin.Localization.MSBuild;
+
+namespace Xamarin.MacDev.Tasks
+{
+	public abstract class WriteAppManifestTaskBase : XamarinTask
+	{
+		#region Inputs
+
+		[Required]
+		public string? AppBundleManifest { get; set; }
```

Comment:
This needs to be an Output property of the ITaskItem type, so the file is also created on Windows.

Reply:
Fixed!